Quest System/Unity Demo/Assets/QSystem/Assets/Scripts/QGiver.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace QSystem
+{
+    public class QGiver : MonoBehaviour
+    {
+        [SerializeField] private string Name = "NPC";
+        [SerializeField] private float InteractRadius = 3f;
+        [SerializeField] private List<Quest> Quests;
+
+        private Quest NextQuest;
+        private SphereCollider SphereCollider;
```

Comment:
Fields met camelCase. Attributes zetten we onder elkaar. In dit geval in de regel boven het bijbehorende field.